app/helpers/kadmin/bootstrap_helper.rb

```diff
@@ -34,5 +34,29 @@ def dropdown(prompt, links = [])
 
       return content_tag(:div, button + list, class: 'dropdown', style: 'display: inline-block;')
     end
+
+    def help_assitance(label, required, title, message)
+      # binding.pry
+      label = t(title)
+      label = label.html_safe
+      
+      message = t(message)
+      message = message.gsub('"', '\"')
+      message = message.gsub("'", "\'")
+      html_message = message.html_safe
+
```

Comment:
This file seems to contain trailing whitespaces. I'd recommend to highlight these in your editor or have rubocop running on your files before commiting.